.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: Java CI build and test
```

There are no test, though. 😬

```suggestion
name: Java Build
```